var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -29,6 +29,8 @@ class Dealii(Package):
     depends_on ("boost@1.59.0:+mpi", when='+mpi')
     depends_on ("bzip2")
     depends_on ("cmake")
+    depends_on ("gls", when='@8.5.0:')
```

Comment:
yes, it is merged. To simplify build i though to make it unconditional dependency as `gsl` does not take too much time to build anyway. But I can also introduce a `variant` if you think it's more appropriate.